asciidoc/courses/importing-data/modules/2-using-data-importer/lessons/1-overview/lesson.adoc

```diff
@@ -12,10 +12,11 @@ video::-QxpQ8FeD9M[youtube,width=560,height=315]
 [.transcript]
 == What is the Neo4j Data Importer?
 
-The Neo4j Data Importer can be found link:https://data-importer.graphapp.io/?acceptTerms=true[here^]:
+For this course, we use Neo4j Data Importer 7.0.
+The Neo4j Data Importer can be found link:https://data-importer.neo4j.io/versions/0.7.0/?acceptTerms=true[here^]:
```

Comment:
Users won't be able to easily connect to local databases on this link as it's neo4j.io, not graphapp.io - just checking that you've already discussed this and are happy with this limitation?

Reply:
I changed it to 0.7.0 - will use  neo4j.io for the hands-on which uses a sandbox for the course.

Will also include info about using latest and graphapp.io.